FWCore/ParameterSet/python/Config.py

```diff
@@ -1431,6 +1442,45 @@ def validate(self):
         #    raise RuntimeError("No input source was found for this process")
         pass
 
+    def handleProcessAccelerators(self, parameterSet):
+        allAccelerators = set([label for acc in self.__dict__['_Process__accelerators'].values() for label in acc.labels()])
+        # 'cpu' accelerator is always implicitly there
+        availableAccelerators = set(["cpu"])
+        for acc in self.__dict__['_Process__accelerators'].values():
+            for l in acc.enabledLabels():
+                availableAccelerators.add(l)
+        unavailableAccelerators = allAccelerators.difference(availableAccelerators)
+        availableAccelerators = sorted(list(availableAccelerators))
+        parameterSet.addVString(False, "@available_accelerators", availableAccelerators)
+
+        # Resolve wildcards
```

Comment:
a curiosity: what is the advantage of resolving the wildcards and the list of selected accelerators at the python level, rather than at the c++ level ?